### PR TITLE
I keep my IDE open and grep for every spelling, punctuation, or grammar error I can find: Part 3

### PR DESCRIPTION
**Pull request by 9600bauds**
leading spaces are the leading cause of brain disease

### CODE REVIEW

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -1,331 +1,331 @@
...
-
-	var/amount_per_transfer_from_this = 10
-	var/possible_transfer_amounts = list(10,25,50,100)
-
-/obj/structure/reagent_dispensers/attackby(obj/item/weapon/W as obj, mob/user as mob)
-	return
-
-/obj/structure/reagent_dispensers/examine(mob/user)
-	..()
+// Assuming this is http://en.wikipedia.org/wiki/Butane
```
Comment (9600bauds) [inline code/modules/reagents/reagent_dispenser.dm:1]: Git went absolutely nuclear trying to deff this file, what the fuck
http://i.imgur.com/S9seAaK.png

**Comment by optimumtact** [comment]
next branch should be i_have_autism

**Comment by 9600bauds** [comment]
development of this game runs on autism alone

**Comment by Intigracy** [comment]
You forgot radio messages doing this.

**Pull request merged**